Cargo.toml

```diff
@@ -249,13 +249,15 @@ harness = false
 name = "comparison_kernels"
 harness = false
 
-[[bench]]
```

Comment:
could you re-activate these, or, what is the issue?

Reply:
I don't find any doc to set env `CARGO_MANIFEST_DIR` in this repo.

If we activate this, it will make other benches fail.

Reply:
isn't `CARGO_MANIFEST_DIR` set automatically by cargo on build?

There shouldn't be interaction between benches (could you give the error?). The `read_parquet` does need some parquet files to be generated (via `python parquet_integration/write_parquet.py`) at runtime, but it should still compile.

Note that it is possible to run only specific benches. E.g. in this case I would use

```
cargo bench --features compute_filter --bench filter_kernels
```

which should be quite fast to compile.

Reply:
`CARGO_MANIFEST_DIR=/tmp cargo bench`  works now.